lib/vf/enumvalue.cpp

```diff
@@ -0,0 +1,51 @@
+/*
+ * Cppcheck - A tool for static C/C++ code analysis
+ * Copyright (C) 2007-2023 Cppcheck team.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "enumvalue.h"
+
+#include "mathlib.h"
+#include "symboldatabase.h"
+#include "token.h"
+#include "valueflow.h"
+
+void ValueFlow::enumValue(SymbolDatabase * symboldatabase, const Settings * settings)
```

Comment:
I think this should be inside `namespace ValueFlow{ ... }`. This will simplify calling the other functions.

Reply:
I had `using namespace ValueFlow;` in the source files which is better since it doesn't increase the indentation. But I went back on it. I guess I add it back.

Reply:
done